app/controllers/dashboard/quick_start_controller.rb

```diff
@@ -57,11 +57,14 @@ def create_experiment_params
     end
 
     def load_project
-      @project = current_team.projects.find_by(id: params.dig(:project, :id))
+      @project = current_team.projects.managable_by_user(current_user).find_by(id: params.dig(:project, :id))
     end
 
     def load_experiment
-      @experiment = @project.experiments.find_by(id: params.dig(:experiment, :id)) if @project
+      return unless @project
```

Comment:
Will it work without experiment?

Reply:
This is for the case where both project and experiment are new, and there are only project params present (project is not in database).